docs/guides/building-a-360-image-gallery.md

```diff
@@ -116,6 +116,9 @@ of the images defined in the [Asset Management System][ams].
   geometry="primitive: plane; height: 1; width: 1"
   material="shader: flat; src: #cubes-thumb"></a-entity>
 ```
+>***Note that the planes will default to position="0 0 0" and since the camera is also in that position by default you will not see them. You can reposition the camera with "wasd". Move the camera in the positive z direction ("S" key if camera is facing the default direction) to see the planes.
```

Comment:
camera is by default in "0 1.6 0" in desktop mode and "0 0 0" in VR mode. https://github.com/aframevr/aframe/blob/master/docs/primitives/a-camera.md